scipy/special/_add_newdocs.py

```diff
@@ -6854,16 +6856,13 @@ def add_newdoc(name, doc):
     where :math:`I` is the regularized incomplete beta function, and
     :math:`x = f d_n/(f d_n + d_d)`.
 
-    The computation time required for this routine is proportional to the
-    noncentrality parameter `nc`.  Very large values of this parameter can
-    consume immense computer resources.  This is why the search range is
-    bounded by 10,000.
+    Note that argument order of `ncfdtr` is different from that
+    of `scipy.stats.ncf.cdf`. `f` is the last parameter of `ncfdtr`
```

Comment:
I don't remember the right way to do this, but I think this is the cause of the CI failure. Probably need to specify the role?

Reply:
Had a similar issue lately. I bet that sphinx tries to find a reference to `f`, changing too ``f`` should fix it.

Reply:
Yeah, there is no docs page for `scipy.stats.ncf.cdf`. You can either refer to ``cdf`` of `scipy.stats.ncf` (with double backticks around 'cdf') or just put the whole thing in double backticks.

Reply:
Thanks everyone. Fixed now.